format/FlightSql.proto

```diff
@@ -867,6 +867,69 @@ enum SqlSupportsConvert {
   SQL_CONVERT_VARCHAR = 19;
 }
 
+/*
+ * Represents a request to retrieve information about data type supported ona Flight SQL enabled backend.
+ * Used in the command member of FlightDescriptor for the following RPC calls:
+ *  - GetSchema: return the schema of the query.
+ *  - GetFlightInfo: execute the catalog metadata request.
+ *
+ * The returned schema will be:
+ * <
+ *   type_name: utf8 not null (The name of the data type, for example: VARCHAR, INTEGER, etc),
+ *   data_type: int not null (The SQL data type),
+ *   column_size: int (The maximum column size that the server supports for this data type.
+ *                     For numeric data, this is the maximum precision.
+ *                     For string data, this is the length in characters.
+ *                     For datetime data types, this is the length in characters of the string representation.
+ *                     NULL is returned for data types where column size is not applicable.),
+ *   literal_prefix: utf8 (Character or characters used to prefix a literal, NULL is returned for
+ *                         data types where a literal prefix is not applicable.),
+ *   literal_suffix: utf8 (Character or characters used to terminate a literal,
+ *                         NULL is returned for data types where a literal suffix is not applicable.),
+ *   create_params: utf8 (A list of keywords, separated by commas, corresponding
+ *                        to each parameter that the application may specify in parentheses
+ *                        when using the name that is returned in the TYPE_NAME field.
+ *                        NULL is returned if there are no parameters for the data type definition.),
+ *   nullable: int not null (Shows if the data type accepts a NULL value),
```

Comment:
I think I might have asked this before but why is this not a bool?

Reply:
(And ditto for `searchable` below)

Reply:
We are basing ourselves in the specs from the JDBC and ODBC. And both parameters in the specs are numeric values

Reply:
I see, the spec has a set of defined values for this, can we create enums for these fields then?

Reply:
Enum created 

Fixed

Reply:
Thanks. Can we explicitly reference the corresponding enum in the description? (And ditto for other such fields.)